service-runner.js

```diff
@@ -69,13 +69,13 @@ ServiceRunner.prototype.run = function run(conf) {
 
         // Set up the logger
         if (!config.logging.name) {
-            config.logging.name = name;
+            config.logging.name = (config.services && config.services.name) || name;
         }
         self._logger = new Logger(config.logging);
 
         // And the statsd client
         if (!config.metrics.name) {
-            config.metrics.name = name;
+            config.metrics.name = (config.services && config.services.name) || name;
```

Comment:
Idem as above, with the distinction that `config.metrics` is a shared object, so a new metrics object should be created for each service (in [`_runWorker()`](https://github.com/wikimedia/service-runner/blob/master/service-runner.js#L278-L285))